GDJS/Runtime/runtimeobject.js

```diff
@@ -337,6 +338,10 @@ gdjs.RuntimeObject.prototype.getRendererObject = function() {
     return undefined;
 };
 
+gdjs.RuntimeObject.prototype.getNormalMapRendererObject = function() {
+    return undefined;
```

Comment:
ubernit: I prefer to return `null` because undefined can mean "you accessed a property that is not even defined, surely something wrong" while `null` is explictly meaning "this is representing a valid value, but this value is null, i.e: there is no normal map object here".